### ISSUE TITLE
redesign chat page.

### ISSUE THREAD

**Issue by ShreyTanna29** [ShreyTanna29/curious.AI#30]
**Is your feature request related to a problem? Please describe.**
chat page is good but can be done more in it

**Describe the solution you'd like**
can make it one like chat gpt has.

**Describe alternatives you've considered**
if you have any other idea or design in mind, go ahead with it, make sure its a modern one, easy to use. 

**Check if applicable**
- [x] I have checked that this feature request does not exists on issues page.

**Additional context**
you will need to make some tweeks or changes to it, no need to replicate as it is.
make sure whenever a user chats with ai, the input bar should go to bottom, it should be on middle only when there are no messages.

![Screenshot from 2024-11-24 00-07-15](https://github.com/user-attachments/assets/4942bf47-141f-45dc-baa1-10e9b50ace30)

**Comment by goutham03062001** [comment]
Hi @ShreyTanna29 is this issue still open? I would like to contribute on this, please assigned if not yet assigned to others

**Comment by ShreyTanna29** [comment]
@goutham03062001 yes, it is open, go ahead and do it, if you need any help then let me know.

**Comment by ShreyTanna29** [comment]
hey @goutham03062001 , are you working on it ? any updates ?
